build.gradle

```diff
@@ -103,27 +113,25 @@ jacoco {
 
 dependencies {
 	compileOnly "com.google.code.findbugs:jsr305:$googleJsr305Version"
-	compile 'org.springframework.integration:spring-integration-core'
-	compile "org.springframework.kafka:spring-kafka:$springKafkaVersion"
-
-	testCompile "org.springframework.kafka:spring-kafka-test:$springKafkaVersion"
-	testCompile 'org.springframework.integration:spring-integration-test'
-	testCompile "com.willowtreeapps.assertk:assertk-jvm:$assertkVersion"
-	testCompile 'org.jetbrains.kotlin:kotlin-reflect'
-	testCompile 'org.jetbrains.kotlin:kotlin-stdlib-jdk8'
-	testCompile 'org.junit.jupiter:junit-jupiter-api'
+	api 'org.springframework.integration:spring-integration-core'
+	api "org.springframework.kafka:spring-kafka:$springKafkaVersion"
 
-	// temporary - gradle bug prevents getting this transitively
-	testCompile "org.apache.kafka:kafka_$scalaVersion:$kafkaVersion:test"
+	testImplementation "org.springframework.kafka:spring-kafka-test:$springKafkaVersion"
+	testImplementation 'org.springframework.integration:spring-integration-test'
+	testImplementation "org.springframework.integration:spring-integration-kotlin-dsl:$springIntegrationKotlinVersion"
```

Comment:
I thought we have fixed the master to be based on the SI Core instead of this extra artifact...

Reply:
Hmmm - not sure how that happened.